build_release_candidate.sh

```diff
@@ -0,0 +1,11 @@
+# Checks if there has been a new version change, and builds the plugin for release if there has.
```

Comment:
Is this executed as a binary under the hood or sourced? `ci.yml` makes it looks like it's the the former and if so we should probs put a shebang at the start. 


inary executables. If you source in another script, then the shebang is ignored.

On the other hand. IF a script is supposed to be sourced, then it is convention to NOT put any shebang at the start.
not sure how this is executed under the hood but we might benefit from a shebang.

Reply:
Yes - good catch!